interp/src/flatten/flat_ir/control/translator.rs

```diff
@@ -612,6 +612,9 @@ impl FlattenTree for cir::Control {
             cir::Control::Static(_) => {
                 todo!("The interpreter does not support static control yet")
             }
+            cir::Control::Repeat(_) => {
+                todo!("The interpreter does not support repeat yet")
```

Comment:
@EclecticGriffin should we make an issue for this? Seems like it would be a "good first issue" since the logic for loops is already there

Reply:
Yeah if you wouldn't mind opening an issue for it! I don't think it would be hard to implement in the existing infra. The flat stuff is it's own other world not worth considering yet